### PR TITLE
(maint) Remove ruby requirement for creating certs

### PR DESCRIPTION
**Pull request by cyberious**
- Add install_ca_certs for windows
- Install from forge instead of git

### CODE REVIEW

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -1,34 +1,26 @@
 require 'beaker-rspec/spec_helper'
 require 'beaker-rspec/helpers/serverspec'
 require 'beaker/puppet_install_helper'
+require 'pry'
```
Comment (tphoney) [inline spec/spec_helper_acceptance.rb]: Does pry need to be in the Gemfile ?

Comment (cyberious) [inline spec/spec_helper_acceptance.rb]: Removed

File: Gemfile [hunk 2]
```diff
@@ -28,7 +28,7 @@ group :system_tests do
     gem 'beaker-rspec',  :require => false
   end
   gem 'serverspec',    :require => false
-  gem 'beaker-puppet_install_helper', :require => false
+  gem 'beaker-puppet_install_helper', '~> 0.3', :require => false
```
Comment (mhaskel) [inline Gemfile]: Does this need to be bound this tightly?

Comment (cyberious) [inline Gemfile]: it needs to be at least 0.3 as that is when we added install_ca_certs

Comment (mhaskel) [inline Gemfile]: Yes, but seems like >= 0.3.0 would cause fewer future issues.

Comment (cyberious) [inline Gemfile]: More so it protects us from what may happen in 1.0 but anything up until 1.0

**Pull request merged**